README.rst

```diff
@@ -66,16 +66,7 @@ http://nilearn.github.io/introduction.html#installation.
 Development
 ===========
 
-Code
-----
+The project is hosted on https://github.com/nilearn/nilearn
```

Comment:
This README.rst will be mostly looked at on github, not sure that this line is worth it.

Then just say "Detailed instructions ..."